apps/ledger-live-desktop/src/renderer/components/SyncOnboarding/Manual/SyncOnboardingCompanion.tsx

```diff
@@ -351,14 +361,18 @@ const SyncOnboardingCompanion: React.FC<SyncOnboardingCompanionProps> = ({
     if (deviceOnboardingState?.isOnboarded && !seededDeviceHandled.current) {
       if (deviceOnboardingState?.currentOnboardingStep === DeviceOnboardingStep.Ready) {
         // device was just seeded
-        setStepKey(StepKey.Backup);
+        setStepKey(recoverUpsellRedirection?.enabled ? StepKey.Apps : StepKey.Backup);
         seededDeviceHandled.current = true;
         return;
       } else if (
         deviceOnboardingState?.currentOnboardingStep === DeviceOnboardingStep.WelcomeScreen1
       ) {
         // switch to the apps step
-        __DEV__ ? setStepKey(StepKey.Backup) : setStepKey(StepKey.Apps); // for ease of testing in dev mode without having to reset the device
+        if (recoverUpsellRedirection?.enabled) {
+          setStepKey(StepKey.Apps);
+        } else {
+          __DEV__ ? setStepKey(StepKey.Backup) : setStepKey(StepKey.Apps); // for ease of testing in dev mode without having to reset the device
```

Comment:
This can be removed now I think

Reply:
When the FF has been removed ?

Reply:
I haven't found out how to test the backup step with this feature flag, so keep it as before for now.